charts/secrets-store-csi-driver/templates/secrets-store.csi.x-k8s.io_secretproviderclasses.yaml

```diff
@@ -61,6 +61,11 @@ spec:
                           type: string
                       type: object
                     type: array
+                  labels:
```

Comment:
Can you revert the changes from `charts` and `deploy` dir. The CRD spec needs to be updated in `manifest_staging/deploy` and `manifest_staging/charts/secrets-store-csi-driver` only. The e2e tests are run against the helm charts in `manifests_staging` - https://github.com/kubernetes-sigs/secrets-store-csi-driver/tree/master/manifest_staging/charts/secrets-store-csi-driver